documentation/src/docs/controller-service.md

```diff
@@ -517,275 +494,253 @@ the lock on the resource and hence permission to process the task. The
 parameters for executing a task are serialized and stored under the
 resource.
```

Comment:
There is a typo in the beginning of this paragraph: `Note that, upon...`.

Reply:
Accepted suggestion